spec/models/course/achievement_ability_spec.rb

```diff
@@ -30,7 +30,7 @@
       end
     end
 
-    context 'when the user is a Course Staff' do
+    context 'when the user is a Course Teaching Staff' do
       let(:course_user) { create(:course_manager, course: course) }
```

Comment:
Should we use the more restrictive role of teaching_assistant here?

I'm thinking we want to test that permission changes don't accidentally lock out the most restrictive staff role, rather than using the manager role which pretty much gets `:manage` defined in most places.

Reply:
Ok will fix the old specs.